Overview.md

```diff
@@ -0,0 +1,114 @@
+computation and filtering. Processing may also involve distribution or storage of the information in a raw or transformed fashion.
+Event Hub consumers are often robust and high-scale platform infrastructure parts with built-in analytics capabilities, like Azure 
+Stream Analytics, Apache Spark, or Apache Storm.   
+   
+Most applications will act either as an event publisher or an event consumer, but rarely both. The exception are event 
+consumers that filter and/or transform event streams and then forward them on to another Event Hub; an example for such a consumer
+and (re-)publisher is Azure Stream Analytics. 
+
+We'll therefore only give a glimpse at publishing and receiving here in this overview and provide further detail in 
+the [Publishing Events](PublishingEvents.md) and [Consuming Events](ConsumingEvents.md) guides. 
+
+### Publishing Events
+
+The vast majority of Event Hub applications using this and other client libraries are and will be event publishers. 
+And for most of these publishers, publishing events is extremely simple. 
+
+With your Java application referencing this client library,	
+which is quite simple in a Maven build [as we explain in the guide](PublishingEvents.md), you'll need to import the 
+*com.microsoft.azure.eventhubs* package with the *EventData* and *EventHubClient* classes.  
+ 
+ 
+```Java
+    import com.microsoft.azure.eventhubs.*;
+```        
+
+Using an Event Hub connection string, which holds all required connection information, including an authorization key or token, 
+you then create an *EventHubClient* instance, which manages a secure AMQP 1.0 connection to the Event Hub.   
+   
+```Java
+    final String namespaceName = "----ServiceBusNamespaceName-----";
+    final String eventHubName = "----EventHubName-----";
+    final String sasKeyName = "-----SharedAccessSignatureKeyName-----";
+    final String sasKey = "---SharedAccessSignatureKey----";
+    ConnectionStringBuilder connStr = new ConnectionStringBuilder(namespaceName, eventHubName, sasKeyName, sasKey);
+		
+    EventHubClient ehClient = EventHubClient.createFromConnectionStringSync(connStr.toString());
+```
+
+Once you have the client in hands, you can package any arbitrary payload as a plain array of bytes and send it. 
+
+```Java
+    EventData sendEvent = new EventData(payloadBytes);
+    ehClient.sendSync(sendEvent);
+```
+         
+The entire client API is built for Java 8's concurrent task model, generally returning 
+[*CompletableFuture<T>*](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html), so the library has these methods suffixed with *Sync* as their Synchronous counterparts/varaints.
+
+Learn more about publishing events, including advanced options, and when you should and shouldn't use those options, 
+[in the event publisher guide](PublishingEvents.md).
+
+### Consuming Events
```

Comment:
Same goes for consumer/consuming and receiver/receiving. Not sure if it's overkill to stick to one word for each but I think that level of strictness will improve clarity in our docs.